fixtures/cameo/q-spot-40-cw.json

```diff
@@ -1,20 +1,17 @@
 {
   "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
   "name": "Q-Spot 40 CW",
-  "shortName": "q-spot-40-cw",
+  "shortName": "CLQS40CW",
   "categories": ["Dimmer"],
   "meta": {
     "authors": ["Nikita"],
-    "createDate": "2024-09-04",
-    "lastModifyDate": "2024-09-04"
+    "createDate": "2024-10-02",
+    "lastModifyDate": "2024-10-02"
   },
-  "comment": "Available in black and white housing",
+  "comment": "Available in black and white housing (WH suffix), not sold anymore.",
   "links": {
     "manual": [
-      "https://www.cameolight.com/en/downloads/file/id/-2054555622"
-    ],
-    "productPage": [
-      "https://www.cameolight.com/en/series/q-spot-series/18002/q-spot-40-ww"
```

Comment:
Why remove the product page link? Is it a different fixture?

Reply:
@FloEdelmann Yes, it's a different fixture. This PR is about `Q-Spot 40 CW`, whereas the product page was for `Q-Spot 40 WW`.
There isn't a matching product page available, because it seems that the cold white variant isn't sold anymore.